frontend/src/Components/Map/MonthFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+
+interface Props {
+  /**
+   * date: date input
+   * onStartChange: function taking argument "date" for Start Date
+   * onEndChange: function taking argument "date" for End date
+   */
+
+  onStartChange: (date: any) => void;
+  onEndChange: (date: any) => void;
+}
+
+/**
+ * Component rendering Month Filter
+ */
+
+const MonthFilter: React.FC<Props> = ({ onEndChange, onStartChange }) => {
+  const [startDate, setStartDate] = useState(new Date('2022/06/08'));
+  const [endDate, setEndDate] = useState(new Date());
+
+  return (
+    <>
+      <DatePicker
+        selected={startDate}
+        onSelect={(date: any) => setStartDate(date)}
+        onChange={onStartChange}
+        selectsStart
+        startDate={startDate}
+        endDate={endDate}
+        minDate={new Date('2022/06/08')}
+        maxDate={new Date()}
+        dateFormat="MM/yyyy"
+        showMonthYearPicker
+      />
+      <DatePicker
+        selected={endDate}
+        onSelect={(date: any) => setEndDate(date)}
+        onChange={onEndChange}
+        selectsEnd
+        startDate={startDate}
+        endDate={endDate}
+        minDate={new Date('2022/06/08')}
+        maxDate={new Date()}
+        dateFormat="MM/yyyy"
+        showMonthYearPicker
+      />
```

Comment:
Why not use `selectsRange` option?

Reply:
Tried that at first but it led to a lot of issues with not being able to pick the dates at the same time, so you would anyway have to click twice. First to select start date, and then again for the end date.

Reply:
Okay perfect :-)